### PR TITLE
[Fix] Compitable with django 1.11

### PR DESCRIPTION
**Pull request by nazmulhasan85**
### Thanks for contributing!

Please make sure all the tests pass, before creating a pull-request.

### CODE REVIEW

File: tests/test_blocks.py [hunk 1]
```diff
@@ -86,5 +86,5 @@ def test_get_context_without_parent_context(self, tweets):
             'limit': 3
         }
 
-        context = block.get_context(value)
+        context = block.get_context(value, parent_context=None)
```
Comment (jjanssen) [inline tests/test_blocks.py]: Should be ``None`` by default, so only the value is sufficient.

**Comment by jjanssen** [comment]
Anything to add @OktayAltay ?

File: tox.ini [hunk 2]
```diff
@@ -1,22 +1,21 @@
 [tox]
 skip_missing_interpreters = True
 skipsdist = True
-usedevelop = True
 
 envlist =
-    py{27,33,34,35}-dj{18,19,110}-wt16
+    py{27,33,34,35}-dj{18,19,110}-wt{18,19,110,111}
```
Comment (OktayAltay) [inline tox.ini]: Can you add dj111 to the envlist because it is defined below but not being used to test.

Comment (OktayAltay) [inline tox.ini]: Can you add dj111 to the envlist because it is defined below but not being used to test.

**Comment by nazmulhasan85** [comment]
@jjanssen Do we need to support wagtail 1.8 or we can skip that .. Just wanted to know.

**Comment by jjanssen** [comment]
1.8 since its the current LTS.

**Comment by nazmulhasan85** [comment]
When executing the `test_get_context_with_parent_context` with wagtail 1.8 its failing. because `get_context` function implemented there with only one parameter but i am calling that with two parameter.

File: tox.ini (hunk 2 above)
Comment (nazmulhasan85) [inline tox.ini]: @OktayAltay, Now i can guess how the was passing with wagtail 1.8 that time. we defined `dj111 ` but not being used then all tests are using `wagtail 1.11` by default and tests are passing successfully.

Comment (nazmulhasan85) [inline tox.ini]: FYI: Wagtailformblock package, for running the tests its always using wagtial 1.11  [Wagtailformblock](https://github.com/LUKKIEN/wagtailformblocks/commit/6effb79a04762bca940de0f478454ec8e6916f5e)

**Comment by nazmulhasan85** [comment]
@jjanssen  Can you accept this merge request. there is a dependency with hendrix website. so we can be aligned with that.

**Pull request merged**